oldcat/chains.json

```diff
@@ -40,6 +40,9 @@
     {
       "name": "bitsong",
       "address": "bitsongvaloper1pqlxe507avzg2hq8ylcju037ywyh3myfsv9wln",
+      "autostake": {
```

Comment:
The `autostake` config goes in your `networks.local.json` file, not in validator registry. It will have no effect here

Reply:
okay. thanks, sorry for bothering.